sale_force_invoiced/model/sale_order_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.depends("order_id.force_invoiced")
+    def _compute_invoice_status(self):
+        for line in self:
+            if line.order_id and line.order_id.force_invoiced:
+                line.update({"invoice_status": "invoiced"})
+            else:
+                return super()._compute_invoice_status()
```

Comment:
The code is incorrect, as it's returning early and doing the super for all the records.